docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -273,7 +266,7 @@ Mapping character types
 ^^^^^^^^^^^^^^^^^^^^^^^
 
 Trino's ``VARCHAR(n)`` maps to ``VARCHAR2(n CHAR)`` if ``n`` is no greater than
-4000. A larger or unbounded ``VARCHAR`` maps to ``NCLOB``.
+1.    A larger or unbounded ``VARCHAR`` maps to ``NCLOB``.
```

Comment:
This seems like an unrelated and incorrect change

Reply:
Wow .. that was so weird. Visual Code thought this is the start of a list and automatically made it the 1. point.. fixed now